recipes/miktex-portable/bld.bat

```diff
@@ -0,0 +1,16 @@
+
+
+7za x miktex-portable-%PKG_VERSION%.exe -o%LIBRARY_PREFIX%\miktex-portable
+if errorlevel 1 exit 1
+
+
+rem SCRIPTS dir should already be created by 7za install
+mkdir "%SCRIPTS%"
+if errorlevel 1 exit 1
+
+rem latex tools must be run from miktex tree
+for %%f in ("%LIBRARY_PREFIX%\miktex-portable\miktex\bin\*.exe") do (
+	echo @%%~dp0\..\Library\miktex-portable\miktex\bin\%%~nf %%* >> "%SCRIPTS%\%%~nf.bat"
```

Comment:
One argument for relative path in the batch files is that you can copy a env enround... But I'm not sure if that is actually a possibility with the rest of the conda environment.

[I'm still pondering how to migrate my y27 conda root to py3.5 and one way would be to copy all my envs to a new miniconda installation and then remove the old path from PATH]